### DISCUSSION TITLE
Auto arrange windows in to preset configurations

### DISCUSSION THREAD

**Discussion by zerst-au** [rxhanson/Rectangle#823] [Ideas]
There are many times when I have many windows all over the place and although I can click each one and then pin it to different locations on the screen ti would also be handy to have an auto arrange feature that automatically does this. If I want to change it after that is no issue at all, but most the time I think I would be happy with the result. 

Let me know your thoughts everyone.

**Comment by rxhanson** [comment]
Configuring a layout and restoring it is a feature in Rectangle Pro (paid), and it isn't really something that I have desire to add into Rectangle, since it is purposefully slim/simple. 

If you're looking for auto-tiling (like Linux WMs), that's not in Rectangle Pro at this time, but I do have a backlog item for it that I will hopefully get to at some point.

To save a layout in Rectangle Pro, you can select "Save Current Layout..." in the menu, or go to the Applications tab to configure layouts (Application groups).

**Reply by zerst-au** [reply]
Thanks @rxhanson. Apologies I have only jsut started using rectangle. Thanks for your reply!

**Reply by AnthonyPoschen** [reply]
as an FYI @rxhanson i came here to search through the discussions to find this ( haven't installed the slim version and was going to straight up buy the pro ) would be good to better list your features on the website for power users so its more obvious what is in the pro so you make more money, like more YT video's would work. Seems like a cool app keep it up

**Reply by gregoryorton-ws** [reply]
I guess the point is that even in rectangle pro, the layout is by "app". I'd love to see it just with some parts of the screen and then you can move apps between those "containers". It's not quite TWMs, but almost, because I can it custom to auto layout like that. I've tried aerospace, and you can't set up dynamic layouts like this. So I'll keep asking everyone until I find someone to do it :)

**Reply by rxhanson** [reply]
@gregoryorton-ws what you're describing is part of the vision that I have for Rectangle Pro. Of course, this was many years ago that I started thinking about this and it hasn't made it in. Good to know that there's still interest; I'd still like to make it.

**Reply by ortonomy** [reply]
yeah, the aerospace experience is nice, but it's really not quite there for me. It has got to have per monitor workspaces for me to be happy with it. I love and pay for rectangle pro, so I'd pay again if you did it, haha.
